### PR TITLE
Change style notation to string

### PR DESCRIPTION
**Pull request by marians**
As we agreed on using the string notification for styles, this PR adapts some styles.

### CODE REVIEW

**Comment by marians** [comment]
Any takers? @ograu @oponder

File: src/components/UI/app_container.js [hunk 1]
```diff
@@ -3,109 +3,111 @@ import PropTypes from 'prop-types';
...
+  background-color: #2f556a;
+  position: relative;
+  cursor: pointer;
+  overflow: hidden;
+  &:hover {
+    z-index: 10;
+    box-shadow: 0px 0px 5px #549ac3;
+    div:nth-of-type(1) {
+      opacity: 1;
+      box-sshadow: 0px 0px 5px #549ac3;
```
Comment (oponder) [inline src/components/UI/app_container.js]:
```suggestion
      box-shadow: 0px 0px 5px #549ac3;
```

File: src/components/app_catalog/detail/install_app_form.js [hunk 2]
```diff
@@ -4,10 +4,10 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import styled from '@emotion/styled';
 
-const FormWrapper = styled.div({
-  display: 'flex',
-  flexDirection: 'column',
-});
+const FormWrapper = styled.div`
+  display: flex;
+  flexdirection: column;
```
Comment (oponder) [inline src/components/app_catalog/detail/install_app_form.js]:
```suggestion
  flex-direction: column;
```

**Comment by marians** [comment]
@oponder How can I find the stylelint complaints?

File: src/components/UI/app_container.js (hunk 1 above)
Comment (marians) [inline src/components/UI/app_container.js]:
```suggestion
      box-shadow: 0px 0px 5px #549ac3;
```

Comment (marians) [reply src/components/UI/app_container.js]: done

**Comment by oponder** [comment]
@marians what I did quickly was merge the `css-lint` branch into this, then `yarn` and then `yarn lint:css`

There's a few more things it uncovered, not due to your changes, but thanks to you making them 'visible' by applying the style that the linter can actually read.

I was doing a bit of digging to understand what was going on, but here is the full output against this branch:

```
src/components/UI/app_container.js
 48:7  ✖  Unexpected unknown property "box-sshadow"   property-no-unknown

src/components/UI/cluster_id_label.js
 21:5  ✖  Unexpected unknown property "margin-seft"   property-no-unknown

src/components/UI/refreshable_label.js
 27:25  ✖  Unexpected empty block   block-no-empty

src/components/UI/navigation/organization_dropdown.js
 104:5  ✖  Unexpected duplicate selector ".selector15 .open .dropdown-toggle.btn-default", first used at line 33   no-duplicate-selectors

src/components/app_catalog/detail/cluster_picker.js
 30:4  ✖  Unknown word   CssSyntaxError

src/components/app_catalog/detail/fileinput.js
 10:4  ✖  Unknown word   CssSyntaxError

src/components/app_catalog/detail/install_app_form.js
 9:3  ✖  Unexpected unknown property "flexdirection"   property-no-unknown
```

**Pull request merged**